### ISSUE TITLE
charset attribute is obsolete for link tag - fix samples

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#47]
```
Reported at 
https://groups.google.com/forum/#!topic/eclipse-plugin-for-phonegap/VAkpiofYlNk
```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 9 Nov 2011 at 10:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 15 Dec 2011 at 10:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 15 Dec 2011 at 10:24
- Added labels: **Milestone-Release1.2.6**

**Comment by GoogleCodeExporter** [comment]
```
The charset="utf-8" in link of css scripts in index.html comes from phonegap's 
sample index.html.

Probably got carried into our demos from there - dates back to at least 
phonegap 0.9.6.
```

Original comment by `libby.ba...@gmail.com` on 20 Dec 2011 at 1:03

**Comment by GoogleCodeExporter** [comment]
```
Fixes available with 1.2.6
```

Original comment by `paul.beu...@gmail.com` on 21 Dec 2011 at 3:57
- Changed state: **Fixed**
